sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/implementation/directconnectivity/rntbd/RntbdClientChannelPool.java

```diff
@@ -1167,10 +1164,8 @@ private void notifyChannelConnect(final ChannelFuture future, final Promise<Chan
                 }
 
                 if (promise.trySuccess(channel)) {
-
-                    if (logger.isDebugEnabled()) {
-                        logger.debug("established a channel local {}, remote {}", channel.localAddress(), channel.remoteAddress());
-                    }
+  
```

Comment:
This one IMO should also stay debug - but in this case I can live with info if you really think it is necessary